public/kibi_timeline_vis_controller.js

```diff
@@ -70,6 +70,10 @@ define(function (require) {
           $scope.visOptions.groups = [];
           _.each(savedSearchesRes, function ({ savedSearch, groups }, i) {
             for (const group of groups) {
+              // get parent field if multifield
+              // because doc_values feature is not applied to string fields
+              group.labelField = group.labelField.length > 0 ? group.labelField.split('.')[0] : group.labelField;
```

Comment:
Do you mean I have to add a new method `fielddata_fields` to the searchsource class here kibi-internal/src/ui/public/courier/data_source/search_source.js ?